libs/application-generic/src/utils/buildBridgeEndpointUrl.ts

```diff
@@ -0,0 +1,22 @@
+import { createHash } from 'crypto';
+
+/*
+ * Creates a bridge endpoint url to be used for request from novu cloud to the local
+ * workflow definition
+ */
+export const buildBridgeEndpointUrl = (
+  apiKey: string,
+  baseAddress: string
+): string => {
+  return `${buildBridgeSubdomain(apiKey)}.${baseAddress}`;
+};
+
+/*
+ * Creates a bridge subdomain based on the apiKey provided. This function is used in several
+ * places, including packages/create-novu-app/templates/index.ts when generating the
+ * subdomain in the bridge application. Developers should take care to keep changes
+ * in sync.
+ */
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** ‏Nice comment 🙏